device/accton/x86_64-accton_as4630_54npe-r0/pmon_daemon_control.json

```diff
@@ -0,0 +1,4 @@
+{
+    "skip_ledd": true,
+    "skip_pcied": true
```

Comment:
Why do you skip pcied for this platform?

Reply:
There is no pcie devices, so we skip this daemon.